src/main/java/org/opentripplanner/routing/edgetype/StreetTransitEntityLink.java

```diff
@@ -55,13 +58,12 @@ public LineString getGeometry() {
     }
 
     public String getName() {
-        return this.stopVertex.getStop().getName();
+        return this.transitEntityVertex.getName();
```

Comment:
The implementation differs between `TransitEntranceLink` and `StreetTransitLink`.

Since `Vertex#getName()` calls `entrance.getName()` or `stop.getName()` this may simplified.